.github/workflows/ci.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   changes:
     runs-on: ubuntu-latest
     outputs:
-      backend: ${{ steps.filter.outputs.backend }}
+      skore: ${{ steps.filter.outputs.skore }}
```

Comment:
`skore-core`? just wondering… I'm anticipating that one day, people will want to install only the "core", and not the "ui". `skore` would then become an umbrella project, composed of both libraries.

Reply:
Let's postpone this use case to when it's needed? It will implies building the skore-ui in a self package.

Reply:
I was just meaning to name the directories, not building things separately. Yes, we can postpone as long as we have this in mind.